cli/common/utils.go

```diff
@@ -238,9 +240,14 @@ Loop:
 	return lastNewLinePos, nil
 }
 
-func ConvertToStringsSlice(s []interface{}) ([]string, error) {
-	stringsSlice := make([]string, len(s))
-	for i, elem := range s {
+func ConvertToStringsSlice(s interface{}) ([]string, error) {
```

Comment:
I think a proper name is "ConvertArrayToStringSlice" but I don't insist

Reply:
In fact, it takes interface argument, not array.

Reply:
Yes, but you can convert only arrays

Reply:
And function returns error if specified interface isn't array. But in fact it converts interfaces.